libraries/ortbConverter/processors/default.js

```diff
@@ -165,3 +169,17 @@ export function setSite(ortbRequest, bidderRequest) {
     ortbRequest.site = Object.assign(getDefinedParams(bidderRequest.refererInfo, ['page', 'domain', 'ref']), ortbRequest.site);
   }
 }
+
+export function onlyOneClientSection(ortbRequest) {
+  ['dooh', 'app', 'site'].reduce((found, section) => {
```

Comment:
If this overall logic will simply drop the latter found objects in the PBS request, is there a desired preference for which objects should be included over the other?  

With this current logic, the order in the `['dooh', 'app', 'site']` array will prefer whatever is listed first and will most likely drop whatever is nearer the end.

I'm not sure if this semi-bias would pose to be an issue in reality, but I thought I would ask just in case.

Reply:
That order is intentional. I know @patmmccann agreed on it but I'm not sure it's been written down anywhere; the idea is that if the pub sets `dooh`, they likely _really_ want that to be sent over, over things that are automatically generated like `site`.

Reply:
Ok thanks for clarifying @dgirardi 

If the conflicts are resolved, I think this would look good.

Reply:
we asked the dooh committee for input on this; this was their preference

Reply:
I added a docs pull here https://github.com/prebid/prebid.github.io/pull/4307/files